lightning/src/chain/chaininterface.rs

```diff
@@ -377,10 +377,19 @@ impl ChainWatchInterface for ChainWatchInterfaceUtil {
 
 	fn filter_block(&self, block: &Block) -> Vec<usize> {
 		let mut matched_index = Vec::new();
+		let mut match_set = HashSet::new();
```

Comment:
It would depend on the context. That naming would apply to inserting into the set, but with membership checks "ancestor" rather than "descendant" would be an accurate naming.

Reply:
Lets not bikeshed a name that will be removed asap.